apps/emqx_resource/i18n/emqx_resource_schema_i18n.conf

```diff
@@ -58,8 +58,8 @@ For bridges only have ingress direction data flow, it can be set to 0 otherwise
 
   start_timeout {
     desc {
-      en: """If 'start_after_created' enabled, how long time do we wait for the resource get started, in milliseconds."""
-      zh: """如果选择了创建后立即启动资源，此选项用来设置等待资源启动的超时时间，单位秒。"""
+      en: """If 'start_after_created' enabled, how long time do we wait for the resource get started."""
```

Comment:
Nit. I suspect that it should be `for the resource to get started` or simply `for the resource to start`.

Reply:
made a complete rephrase.